internal/install/command.go

```diff
@@ -27,7 +28,7 @@ var Command = &cobra.Command{
 	Use:    "install",
 	Short:  "Install New Relic.",
 	PreRun: client.RequireClient,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I dont follow why the error handling is changing here. Is this required to surface an error message to the user when we notice the paywal?

Reply:
We can also handle this at the subcommand level if that is preferred, but this felt like a top-level type of error scenario. Either way we decide is fine with me.